app/bundles/EmailBundle/Entity/Email.php

```diff
@@ -765,6 +772,27 @@ public function setReplyToAddress($replyToAddress)
         return $this;
     }
 
+    /**
+     * @return mixed
+     */
```

Comment:
```suggestion
```
The comments for this and the method above can be removed as the IDEs will use the actual types for type hints.